File: ProductAngular/src/app/model/supplier.ts
export class supplier{
    SupplierId?:number;
    CompanyName?:string;
    ContactName?:string;
    Address?:string;
    Phone?:string;
    Email?: string;
    constructor(
        _SupplierId?:number,
        _CompanyName?:string,
        _ContactName?:string,
        _Address?:string,
        _Phone?:string,
        _Email?: string, 
    )
    {
       this.SupplierId=_SupplierId;
       this.CompanyName=_CompanyName;
       this.ContactName=_ContactName;
       this.Address=_Address;
       this.Phone=_Phone;
       this.Email=_Email;
    }



}

File: ProductAngular/ProjectAPI/ProjectAPI/Models/Supplier.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ProjectAPI.Models
{
    public class Supplier
    {
        [Key]
        public int SupplierId { get; set; }
        
        public string CompanyName { get; set; }
       
        public string ContactName { get; set; }
       
        public string Address { get; set; }
       
        public string Phone { get; set; }
        
        public string Email { get; set; }
        public virtual ICollection<Product> Products { get; set; }
    }
}
